README.rst

```diff
@@ -121,8 +121,8 @@ Is it a URL or a search term?
 Parse URL...
 ------------
 
-Check HSTS list...
```

Comment:
I'm not sure this was a typo... http://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security